docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -1,16 +1,16 @@
-# TODO: come up with a nice-looking generic placeholder icon/label to go here
-metadata:
-  title: |
-    CourtFormsOnline: MassAccess
-  short title: |
-    CourtFormsOnline
-  title url: https://courtformsonline.org  
-  exit url: https://courtformsonline.org
 ---
 objects:
   - al_logo: DAStaticFile.using(filename="ma_logo.png")
 ---
 default screen parts:
+  title: |
+    ${ AL_ORGANIZATION_TITLE }
+  short title: |
+    ${ AL_ORGANIZATION_TITLE }
+  title url: |
+    ${ AL_ORGANIZATION_HOMEPAGE }
+  exit url: |
+    ${ AL_ORGANIZATION_HOMEPAGE }
```

Comment:
```suggestion
  title: |
    ${ all_variables(special='metadata').get('title', AL_ORGANIZATION_TITLE) }
  short title: |
    ${ all_variables(special='metadata').get('short title', AL_ORGANIZATION_TITLE) }
  title url: |
    ${ all_variables(special='metadata').get('title url', AL_ORGANIZATION_HOMEPAGE) }
  exit url: |
    ${ all_variables(special='metadata').get('exit url', AL_ORGANIZATION_HOMEPAGE) }
```

Reply:
This is a little convoluted, but will respect existing `metadata: title` values that are very widespread in existing interviews.